node_modules/oae-activity/tests/test-email.js

```diff
@@ -855,6 +857,59 @@ describe('Activity Email', function() {
     });
 
     /**
+     * Test that verifies that activity emails are not delivered to users who have deleted their
+     * profiles
+     */
+    it('verify email is not delivered to users who have since deleted their profile', function(callback) {
```

Comment:
Ideally, a user would not receive any activities or email notifications through a deleted group being a member of an active group or being a member of a content item. Can you describe the potential performance drawbacks that are involved here?

Regardless, priority of this is definitely lower than inviting guests.